### PR TITLE
Update App.js

### PR DESCRIPTION
**Pull request by pritam-kr**
- [Feat: Login, Video listing, single video page](https://github.com/pritam-kr/playme/pull/5/commits/e8b5b9d08aaac81b5c372c2f18a09f513c1d8130)
-  [Feat: video card](https://github.com/pritam-kr/playme/pull/5/commits/f2adc4aab22704b8be615f8e12a12a868990e9a3)
- [Feat: video context for global data](https://github.com/pritam-kr/playme/pull/5/commits/1eb9a65d723155fb8d16a3155b2e84a4cf6d2ca8)
- created helper function
- created context for video and auth

Screenshot (24)

Screenshot (23)

Screenshot (22)

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,23 +1,37 @@
...
+
+  }
 
   return (
     <div className="app-container">
+
       {pathname !== "/" && <Topbar />}
+      <SideBar />
+      {/* {pathname !== "/login" && <Topbar />} */}
       {/* {pathname !== "/" && <Sidebar />} */}
```
Comment (srishti-maurya) [inline src/App.js]: you may delete the comments if not required

File: src/Context/AuthContext.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+                 console.log(errors)
+            }
+
+        }else if (!regEx.test(formData.email)) {
+             
+        } else {
+             
+        }
+        
+    }
```
Comment (srishti-maurya) [inline src/Context/AuthContext.js:38]: if this is not intentional then you may either want to remove this condition or return something

File: src/Context/SingleVideoContext.js [hunk 3]
```diff
@@ -0,0 +1 @@
+// import { createContext, useContext, useReducer } from "react";
```
Comment (srishti-maurya) [inline src/Context/SingleVideoContext.js]: you might re-check this file if you need it or not

File: src/Pages/Explore/Explore.jsx [hunk 4]
```diff
@@ -1,28 +1,29 @@
...
                 <div className="categories-wrapper">
                     <ul>
-                        <li className="category-list">All</li>
-                        <li className="category-list">Front-end</li>
-                        <li className="category-list">Back-end</li>
-                        <li className="category-list">Game</li>
-                        <li className="category-list">API</li>
-                        <li className="category-list">Reactjs</li>
+                      <li className={getUniqueCategory?.length === 5 ? "category-list active" : "category-list"} onClick={() => dispatch({type: "FILTER_RESET"})}>All</li> 
+                        {getUniqueCategory?.map((eachCategory, i) => <li className="category-list" onClick={() => dispatch({type: "GET_CATEGORY_NAME", payload: eachCategory})} key={i}>{eachCategory}</li>)}
```
Comment (srishti-maurya) [inline src/Pages/Explore/Explore.jsx:20]: Suggestion: you may add a comment to explain or apply prettier here for better readability

**Review by srishti-maurya — APPROVED** [review]
Code looks clean and the comments are descriptive

File: src/Pages/SingleVideo/SingleVideo.css [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+  max-width: 100rem;
+  margin: 0 auto;
+  color: var(--white-text);
+}
+
+.video-frame {
+  padding: 1rem;
+  background: var(--dark-background-color);
+  width: 100%;
+  height: 300px;
```
Comment (palash23jain) [inline src/Pages/SingleVideo/SingleVideo.css:12]: you can use rem to maintain consistency across

**Review by palash23jain — APPROVED** [review]
Code looks clean

File: src/App.js (hunk 1 above)
Comment (pritam-kr) [reply src/App.js]: ok

File: src/Context/AuthContext.js (hunk 2 above)
Comment (pritam-kr) [reply src/Context/AuthContext.js:38]: its intentional.

File: src/Context/SingleVideoContext.js (hunk 3 above)
Comment (pritam-kr) [reply src/Context/SingleVideoContext.js]: ok

File: src/Pages/Explore/Explore.jsx (hunk 4 above)
Comment (pritam-kr) [reply src/Pages/Explore/Explore.jsx:20]: ok

Comment (pritam-kr) [reply src/Pages/Explore/Explore.jsx:20]: Thank you

**Pull request merged**